lib/ui/text.dart

```diff
@@ -80,6 +81,8 @@ class FontWeight {
   /// an [AnimationController].
   static FontWeight lerp(FontWeight a, FontWeight b, double t) {
     assert(t != null);
+    if (a == null && b == null)
+      return null;
```

Comment:
In practice, since LibTxt's TextStyle defaults null font weights to w400, and dart:ui's `FontWeight` and `TextStyle` are only created as the final step before passing the data into native code, this change should cause minimal real-world impact.

The case where this makes a difference is when the user intended the font weight to inherit from a previous TextStyle in the stack. The behavior suggested here would allow it to inherit instead of be set to w400. On the framework side, the inherit flag can also be used to force inherit behavior.

Reply:
Was there any specific case where the old behavior was causing unexpected behavior?

Reply:
I only noticed it because I was experimenting with a change to `BottomNavigationBar` in the framework. Basically I wanted to add the fields: `TextStyle selectedLabelStyle` and `TextStyle unselectedLabelStyle`. Then, the bottom nav would `lerp` between the two text styles so that they would animate properly. 

I noticed in my tests that this was adding a `FontWeight.w400` somewhat unexpectedly, since pretty much every other [field on the TextStyle class] lerps to return null. Like `lerpDouble` and `Color.lerp`.